### PR TITLE
Add Q&A about what sets GrapheneOS apart from LineageOS.

### PR DESCRIPTION
**Pull request by Peter-Easton**
This one might be a tad bit more controversial, so I completely understand if it has to wait for later or not be included at all. I tried to be as positive as I could.

I thought since much of the project has been terribly misunderstood, I could add this.

### CODE REVIEW

File: static/faq.html [hunk 1]
```diff
@@ -180,6 +182,57 @@ <h2 id="when-devices">
...
+            of phones that are no longer supported by their vendors. LineageOS deliberately elects
+            not to utilize Verified Boot, as this is seen as desirable by users that wish to
+            customize their system image by means of having root access. Since many LineageOS users
+            desire functionality that would otherwise be prohibited by the Android Mandatory Access
+            Control policies which are implemented for security and preventing exploits, LineageOS
+            reduces the restrictiveness of these policies.</p>
+
+            <p>LineageOS and GrapheneOS have extremely different and mutually exclusive goals, with
+            LineageOS prioritizing permissiveness and customizeability, and GrapheneOS prioritizing
+            correctenss, verifiability, exploit mitigation, and security. GrapheneOS has never used
```
Comment (inthewaves) [inline static/faq.html]:
```suggestion
            correctness, verifiability, exploit mitigation, and security. GrapheneOS has never used
```

**Comment by Peter-Easton** [comment]
Whoops. For some reason I think the Github WebUI and the version on my computer didn't agree. This should be fixed now, sorry!

**Comment by Peter-Easton** [comment]
I tried trimming it down a little bit to four paragraphs and discarded some of the information that's covered in the new Q&As.

**Comment by thestinger** [comment]
I would prefer not to have this in the FAQ. There's a features page covering what GrapheneOS adds to AOSP now.

**Pull request closed without merging**